Robot account failed to delete the artifact when replication

**Expected behavior and actual behavior:**
Expectation: The artifact can be deleted when using the robot account for replication
Actual: Failed to delete artifact when using robot account for replication 

**Steps to reproduce the problem:**
1 Create a robot account with all permissions on Harbor B 
2 Create harbor B endpoint on Harbor A
3 Create Replication Rule, Replication mode on Harbor A
Push-based, Trigger Mode is Event Based, check Delete remote resources when locally deleted 
4 Push an image to Harbor A to create a Replication task normally, and you can also see the replicated image in Harbor B
5 Delete an image in Harbor A, you can see that the Replication task is created, but the result is failed, and the image in Harbor B is not deleted


**Versions:**

- harbor version: 2.4.1

**Additional context:**

```
2022-01-04T07:16:53Z [INFO] [/controller/replication/transfer/image/transfer.go:125]: client for source registry [type: harbor, URL: http://core:8080, insecure: true] created
2022-01-04T07:16:53Z [INFO] [/controller/replication/transfer/image/transfer.go:135]: client for destination registry [type: harbor, URL: https://xxx.xxx.xxx.xxx, insecure: true] created
2022-01-04T07:16:53Z [ERROR] [/controller/replication/transfer/image/transfer.go:408]: failed to delete the manifest of artifact test01/photon:4.0 on the destination registry: http status code: 401, body: {"errors":[{"code":"UNAUTHORIZED","message":"unauthorized to access repository: test01/photon, action: delete: unauthorized to access repository: test01/photon, action: delete"}]}
```

--- a/src/portal/src/app/base/left-side-nav/system-robot-accounts/system-robot-util.ts
+++ b/src/portal/src/app/base/left-side-nav/system-robot-accounts/system-robot-util.ts
@@ -58,6 +58,11 @@ export const INITIAL_ACCESSES: FrontAccess[] = [
         "checked": true
     },
     {
+        "resource": "repository",
+        "action": "delete",
+        "checked": true
+    },
+    {
         "resource": "artifact",
         "action": "delete",
         "checked": true

--- a/src/portal/src/app/base/project/repository/artifact/artifact-list-page/artifact-list/artifact-list-tab/artifact-list-tab.component.ts
+++ b/src/portal/src/app/base/project/repository/artifact/artifact-list-page/artifact-list/artifact-list-tab/artifact-list-tab.component.ts
@@ -722,6 +722,7 @@ export class ArtifactListTabComponent implements OnInit, OnDestroy {
     retag() {
         if (this.selectedRow && this.selectedRow.length && !this.depth) {
             this.retagDialogOpened = true;
+            this.imageNameInput.imageNameForm.reset();
             this.retagSrcImage = this.repoName + ":" + this.selectedRow[0].digest;
         }
     }

